components/Common/ProgressionSidebar/ProgressionSidebarItem/index.tsx

```diff
@@ -0,0 +1,32 @@
+import classNames from 'classnames';
+import type { FC } from 'react';
+
+import ProgressionSidebarIcon from '@/components/Common/ProgressionSidebar/ProgressionSidebarIcon';
+import { Link } from '@/navigation.mjs';
+
+import styles from './index.module.css';
+
+type ProgressionSidebarItemProps = {
+  url: string;
+  title: string;
+  isActive?: boolean;
+};
+
+const ProgressionSidebarItem: FC<ProgressionSidebarItemProps> = ({
+  url,
+  title,
+  isActive,
+}) => (
+  <li
```

Comment:
Instead of attributing styles on `li` is there any reason to not use `ActiveLink` component? And make the the style be on the `a` element?

Reply:
Updated it with `ActiveLink ` ✨